packages/web3-utils/src/converters.ts

```diff
@@ -551,17 +553,31 @@ export const toWei = (number: Numbers, unit: EtherUnits): string => {
 	if (!denomination) {
 		throw new InvalidUnitError(unit);
 	}
+	let parsedNumber = number;
+	if (typeof parsedNumber === 'number'){
+		if (parsedNumber  < 1e-15){
+			console.warn('Warning: The type `numbers` that are large or contain many decimals may cause loss of precision, it is recommended to use type `string` or `BigInt` when using conversion methods')
+			throw new InvalidNumberDecimalPrecisionLossError(number as number);
```

Comment:
I think it should be either a warning or an exception, but not both.
And could you please double check that the CHANGELOG.md is updated to mention the exception thrown for this case.

Reply:
as well the error is a breaking change, so i removed the error and kept the warning. thanks 👍